instrumentation_sql.go

```diff
@@ -51,13 +53,26 @@ func InstrumentSQLDriver(sensor *Sensor, name string, driver driver.Driver) {
 // SQLOpen is a convenience wrapper for `sql.Open()` to use the instrumented version
 // of a driver previosly registered using `instana.InstrumentSQLDriver()`
 func SQLOpen(driverName, dataSourceName string) (*sql.DB, error) {
+
 	if !strings.HasSuffix(driverName, "_with_instana") {
 		driverName += "_with_instana"
 	}
 
 	return sql.Open(driverName, dataSourceName)
 }
 
+//go:linkname drivers database/sql.drivers
+var drivers map[string]driver.Driver
+
+// SQLInstrumentAndOpen combines `InstrumentSQLDriver` and `SQLOpen` calls
+func SQLInstrumentAndOpen(sensor *Sensor, driverName, dataSourceName string) (*sql.DB, error) {
```

Comment:
Should we update the `http-database-greeter` example with the new code, or create a new one?
As per our agreement of having an `example` folder for all new instrumentations, this would be my preferable choice. 
As the SQL instrumentation is part of the core and we already have an `example` folder anyway, it would be just a matter of deciding if you want to change http-database-greeter or create a new one.

The only additional thing I would like to suggest is to add a docker-compose file into this directory that - initially - starts Postgres as a convenience.

What do you think?

Reply:
Makes sense. I updated an example, please check. It requires one more commit, after the main module will be released.